src/ignitus-Dashboard/ignitus-Dashboard-Header/Components/dashBoardHeader.js

```diff
@@ -40,64 +53,72 @@ class dashBoardHeader extends React.Component {
     }
     return (
       <React.Fragment>
-        <nav className="navbar navbar-expand-lg whitenav">
+        <nav className="navbar navbar-expand-lg whitenav text-blue">
```

Comment:
Please don't use BS4 as I already mentioned this that we are migration to pure `css` then to => `emotion`, but you can make use of pure `css` as of now, if you don't know `emotion`. see #403

Reply:
And while writing pure css, please write classNames in `camelCase`.